google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/OptionTest.java

```diff
@@ -50,14 +46,13 @@ public void testHashCode() {
     assertEquals(OPTION.hashCode(), OPTION_EQUALS.hashCode());
   }
 
-  @Test
+  @Test(expected = NullPointerException.class)
```

Comment:
Google best practice is not to use expected exceptions. As of JUnit 4.13 you can use assertThrows instead.